Lib/ntpath.py

```diff
@@ -523,9 +514,9 @@ def expandvars(path):
                         value = os.fsencode(os.environ[os.fsdecode(var)])
                     else:
                         value = environ[var]
+                    res += value
                 except KeyError:
-                    value = dollar + var
-                res += value
+                    res += dollar + var
```

Comment:
How much does this change speed things up? I weakly prefer the old code, though I don't have a strong preference either way. But if this change doesn't actually have any impact on performance, we should revert it. We should aim for the minimal diff to achieve what we're trying to accomplish.

Reply:
Nothing measureable.